cupy/sparse/construct.py

```diff
@@ -129,3 +131,28 @@ def random(m, n, density=0.01, format='coo', dtype=None,
     vals = data_rvs(k).astype(dtype)
     return cupy.sparse.coo_matrix(
         (vals, (i, j)), shape=(m, n)).asformat(format)
+
+
+def rand(m, n, density=0.01, format='coo', dtype=None, random_state=None):
+    """Generates a random sparse matrix.
```

Comment:
Could you add more information about this function here? For example, what distribution is used to sample values?